assets/js/components/read-more/index.js

```diff
@@ -32,7 +44,22 @@ class ReadMore extends Component {
 
 	componentDidMount() {
 		if ( this.props.children ) {
-			this.lineHeight = this.element.current.clientHeight + 1;
+			const { maxLines } = this.props;
+			const lineHeight = this.reviewSummary.current.clientHeight + 1;
+			const reviewHeight = this.reviewContent.current.clientHeight + 1;
+			const maxHeight = ( lineHeight * maxLines ) + 1;
+			const isClamped = reviewHeight > maxHeight;
```

Comment:
The name of the variable seems a bit awkward to me here.  Where it is used, it appears this is used to trigger when the content _needs_ to be clamped - i.e. it appears if `this.state.isClamped === true`, then the content needs to be clamped, otherwise it doesn't need clamped - so `isClamped` doesn't really mean it's clamped yet?.   I get that in the `render` function it is used to indicate whether to just return the full content or not.

So I'm wondering if it should be named something like `shouldClamp`?

Reply:
Maybe `enabled` - Ill change it, it was confusing when building too :p